src/components/experiment-tracking/run-metadata/index.js

```diff
@@ -15,20 +15,20 @@ const RunMetadata = ({ isSingleRun, runs = [] }) => {
     setToggleNotes({ ...toggleNotes, [index]: !toggleNotes[index] });
   };
 
-  return (
+  return runs.length === 0 ? (
+    <div>Loading...</div>
```

Comment:
Maybe this `div` should have the same class names as the one below so it receives the same spacing?
```suggestion
    <div className={classnames('details-metadata', {
        'details-metadata--single': isSingleRun,
      })}>Loading...</div>
```